documentation/modules/auxiliary/admin/smb/ms17_010_psexec.md

```diff
@@ -0,0 +1,84 @@
+MS17-010 are psexec are two of the most popular exploits against Microsoft Windows. This module bolts the two together.
+
+You can run any command as SYSTEM. Note: unlike EternalBlue, kernel shellcode is not used to stage Meterpreter, so you might have to evade your payloads.
+
+* CVE-2017-0146 (EternalChampion/EternalSynergy) - exploit a race condition with Transaction requests
+* CVE-2017-0143 (EternalRomance/EternalSynergy) - exploit a type confusion between WriteAndX and Transaction requests
+
+This module is highly reliable and preferred over EternalBlue where a Named Pipe is accessible for anonymous logins (generally, everything pre-Vista, and relatively common for domain computers in the wild).
+
+## Vulnerable Server
+
+To be able to use auxiliary/admin/smb/ms17_010_psexec:
```

Comment:
Should this be `auxiliary/admin/smb/ms17_010_command`? See also the documentation filename.

```
msf5 > use auxiliary/admin/smb/ms17_010_psexec
[-] Failed to load module: auxiliary/admin/smb/ms17_010_psexec
```